### ISSUE TITLE
[Bug] Insert xml tag `mark` after pressing enter with `/` command

### ISSUE THREAD

**Issue by waynewaynetsai** [vipzhicheng/logseq-plugin-vim-shortcuts#18]
#### Description

Insert xml tag `mark` after pressing enter with `/` command

#### Screen Shot

1. Search Word with `/` command

 ![image](https://user-images.githubusercontent.com/26406404/176337788-f32cf82f-4efd-425b-80d8-a27353609ed3.png)

2. Pressing enter: unexpected insert xml tag `mark`

![image](https://user-images.githubusercontent.com/26406404/176337859-39914ef4-664a-45ab-b48d-d1b37797954d.png)

**Comment by Zumh** [comment]
Temporary fix > After search with / and enter press / again to toggle the highlight or xml tag. 
This is probably a feature lol! ahhaaa

**Comment by vipzhicheng** [comment]
The search feature is an experimental feature, I still find a better solution for this. For the flaw, I provider a command in Palette (mod+shift+p), "Search Cleanup"

**Comment by vipzhicheng** [comment]
Please upgrade and test if it is solved now.

**Comment by waynewaynetsai** [comment]
Version: v0.1.13
Environment: MacOS Monterey v12.5

I've test this feature with upgrading plugin(v0.1.13).
After upgrading, i can not search text correctly after pressing `enter`. 

Thinks for your reply!


> 請升級並測試現在是否已經解決。

**Comment by vipzhicheng** [comment]
I still do not know when the declaration take effect, but the Logseq Team give me an advice, and it works for me .

Two steps.

1. Uninstall the plugin.
2. Open developer tool which is `cmd+alt+i` on Mac, then run the scripts below in Console.

```
logseq.api.__install_plugin({id:"logseq-vim-shortcuts", repo: "vipzhicheng/logseq-plugin-vim-shortcuts", theme: false, effect: true})
```

**Comment by waynewaynetsai** [comment]
> I still do not know when the declaration take effect, but the Logseq Team give me an advice, and it works for me .
> 
> Two steps.
> 
> 1. Uninstall the plugin.
> 2. Open developer tool which is `cmd+alt+i` on Mac, then run the scripts below in Console.
> 
> ```
> logseq.api.__install_plugin({id:"logseq-vim-shortcuts", repo: "vipzhicheng/logseq-plugin-vim-shortcuts", theme: false, effect: true})
> ```

Thanks for your quick reply!  
I reinstall the plugin with above steps.
And the problem has been fixed at v0.1.14!
